src/udev/meson.build

```diff
@@ -220,19 +220,19 @@ tests += [
                 'base' : test_libudev_base,
         },
         {
-                'sources' : files('test-udev-node.c'),
+                'sources' : files('test-udev-manager.c'),
                 'base' : test_libudev_base,
         },
         {
-                'sources' : files('test-udev-rules.c'),
+                'sources' : files('test-udev-node.c'),
```

Comment:
looks like these renames are in the wrong commit

Reply:
Not really, the source entries are just kept sorted.

Reply:
github getting confused then